recipe/bld.bat

```diff
@@ -1,3 +1,5 @@
+rm -rf C:/hostedtoolcache/windows/Python
```

Comment:
Noticed this removal. FWIW put together a conda-smithy PR ( https://github.com/conda-forge/conda-smithy/pull/1966 ) to cleanup paths like this. Would be interested in your feedback on it. Also would be curious to know if it helps your use case